src/drivers/magnetometer/lis3mdl/lis3mdl.cpp

```diff
@@ -58,14 +58,14 @@ LIS3MDL::LIS3MDL(device::Device *interface, const char *path, enum Rotation rota
 	_continuous_mode_set(false),
 	_mode(CONTINUOUS),
 	_rotation(rotation),
-	_measure_ticks(0),
+	_measure_interval(0),
 	_class_instance(-1),
 	_orb_class_instance(-1),
 	_range_ga(4.0f),
 	_range_scale(0),                // default range scale from counts to gauss */
 	_check_state_cnt(0),
 	_cntl_reg1(
-		CNTL_REG1_DEFAULT),     // 1 11 111 0 0 | temp-en, ultra high performance (XY), fast_odr disabled, self test disabled
+		CNTL_REG1_DEFAULT), // 1 11 111 0 0 | temp-en, ultra high performance (XY), fast_odr disabled, self test disabled
```

Comment:
I think there is a stray newline here, might be a good chance to delete it.

Reply:
Fixed.